android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -135,6 +135,10 @@ private long getBinaryResourcesModifiedTime() {
         }
     }
 
+    public static String getBundleUrl() {
+        return getBundleUrl("index.android.bundle");
```

Comment:
not super important, but may be define `"index.android.bundle"` as a static field above.